pkg/controller/controllercmd/builder.go

```diff
@@ -142,6 +147,11 @@ func (b *ControllerBuilder) WithLeaderElection(leaderElection configv1.LeaderEle
 		return b
 	}
 
+	// Set flag that SNO leader election configs can be used since user provided no timing configs
+	b.canUseSNOLeaderElection = (leaderElection.LeaseDuration.Duration == 0 &&
```

Comment:
This is actually `userExplicitlySetLeaderElectionValues`

Reply:
done, updated logic